DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,7 @@ public function getConfigTreeBuilder()
             ->fixXmlConfig('deployment_name')
             ->children()
                 ->booleanNode('enabled')->defaultTrue()->end()
+                ->booleanNode('twig')->defaultTrue()->end()
```

Comment:
just set the default to `class_exists(\Twig_Environment::class)` or maybe we on't need a config at all

Reply:
I may want to turn this off even though I have Twig installed.

Reply:
So something like this:
```php
->booleanNode('twig')->defaultValue(class_exists(\Twig_Environment::class))->end()
```